exercises/connect/connect.spec.js

```diff
@@ -0,0 +1,110 @@
+import Board from './connect';
+
+describe('Judging a game of connect', () => {
+
+  it("an empty board has no winner", () => {
+    let board = [
+      ". . . . .",
+      " . . . . .",
+      "  . . . . .",
+      "   . . . . .",
+      "    . . . . ."
+    ];
+    expect(new Board(board).winner()).toEqual("");
+  });
+
+  it("X can win on a 1x1 board", () => {
```

Comment:
All tests except the first one should be declared with `xit` so they're skipped by default.  I won't call this out everywhere, but it should be changed for this test and all the remaining ones.